src/app/admin/groupDialogAdd/groupDialogAdd.component.html

```diff
@@ -0,0 +1,48 @@
+<h1 mat-dialog-title>Add new group</h1>
```

Comment:
https://angular.io/guide/styleguide
Regarding to naming conventions component should be named `group-add-dialog`